Elf/__init__.py

```diff
@@ -922,11 +930,29 @@ def isSharedObject(self):
             return True
         return False
 
+    def hasInterpreter(self):
```

Comment:
good to have, and is used by isExecutable (which has been only partially correct for a while now)